examples/multi_window.py

```diff
@@ -7,6 +7,8 @@
 """
 
 def load_html():
+    import time
+    time.sleep(1)
```

Comment:
Then the bug is still on I think. I added the delay because after adding multi-
window support, I occasionally get this:

![screen shot 2017-05-03 at 10 09 17 pm](https://cloud.githubusercontent.com/assets/18247990/25671381/6d35fdd0-304d-11e7-9e9c-0d740e626802.png)

It's just a tiny delay to be on the safe side, can't hurt, can it?